src/unitStats/mappingSbps.ts

```
@@ -283,6 +307,53 @@ const mapExtensions = (root: any, sbps: SbpsType, locale: string = "en") => {
           };
         }
         break;
+
+      case "squad_upgrade_ext":
```

Biome (2.4.15)

[error] Duplicate case label.

(lint/suspicious/noDuplicateCase)